frontend/src/scenes/session-recordings/player/PlayerFrame.tsx

```diff
@@ -28,10 +28,25 @@ export const PlayerFrame = (): JSX.Element => {
             return
         }
 
+        const handleReplayerErrors = (event: ErrorEvent): void => {
+            // sometimes replayer throws errors but playback can continue
+            // we don't want to show an error message in that case
+            // so let's swallow but report replayer errors
```

Comment:
I wonder if the errors can lead to an even more errored replay which makes the experience even worse, maybe we should keep playing but at least give a hint to the user that an error happened.

Reply:
That's a great follow-up idea...

I want to make time to add a dev/debug/doctor view... will fold this in there